RecoTracker/TkSeedGenerator/plugins/BuildFile.xml

```diff
@@ -1,3 +1,4 @@
+<use name="PhysicsTools/TensorFlow"/>
```

Comment:
Is it really *that hard* to add the dependencies in alphabetical order ?

Reply:
@cms-sw/reconstruction-l2 could you add this suggestion to the list of requests you make of people ?

Reply:
I'm really sorry, nobody ever told me this requirement before (probably it is written in some twiki, but I missed it)